.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,14 +28,21 @@ For refactoring and code cleanup changes, exercise the code before and after the
 
 ### Proposed changelog entries
 
-- Entry 1: Issue, human-readable text
-- […]
-
-<!-- Comment:
+<!--
+Please propose a changelog entry (or entries) for this pull request below between the lines with three backticks.
+Add more than one entry if necessary, but be aware that this is usually an indicator that a PR should be split into multiple PRs.
 The changelog entry should be in the imperative mood; e.g., write "do this"/"return that" rather than "does this"/"returns that".
+Use basic HTML like '<strong>', '<a href="…">', '<code>' to format changelog entries. Please do not use Markdown or Asciidoc.
```

Comment:
FWIW backticks are handled for code, https://github.com/jenkinsci/core-changelog-generator/blob/c13ec5065e4c16b836edc15d7d0a2b3ddd22f6ac/generate-jenkins-changelog.rb#L144

it wouldn't be hard to handle markdown links